src/terrain/material.cpp

```diff
@@ -1,21 +1,23 @@
 #include "material.hpp"
 
-namespace terrain{
+namespace terrain {
 
-std::vector<uint32_t> color_ids_map;
-std::unordered_map<uint32_t, uint16_t> colors_inverse_map;
+std::vector<uint32_t> Terrain_Color_Mapping::color_ids_map;
```

Comment:
```suggestion
std::vector<uint32_t> TerrainColorMapping::color_ids_map;
```
PascalCase for classes